Project.toml

```diff
@@ -1,17 +1,15 @@
 name = "VariantVisualization"
 uuid = "7f8bf45c-0ad5-53b5-97e6-33c1305e0aa4"
 authors = ["George Tollefson <gatollefson@gmail.com>"]
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
fix the version and use the registrator bot. Check JuliaRegistrator org on github.